test/python/qpy/test_circuit_load_from_qpy.py

```diff
@@ -35,6 +36,8 @@ def assert_roundtrip_equal(self, circuit):
         new_circuit = load(qpy_file)[0]
 
         self.assertEqual(circuit, new_circuit)
+        if circuit.layout is not None:
+            self.assertEqual(circuit.layout, new_circuit.layout)
```

Comment:
I'm pretty sure this assertion should be valid even if `circuit.layout` _is_ None.

Reply:
That's fair, I think I did this because I didn't want to assert the layout in cases when we were calling `transpile()` and not adding a layout or something like that, but it's been long enough I don't really remember. But, yeah if circuit.layout is None then this should be true too.

```suggestion
        self.assertEqual(circuit, new_circuit)
        self.assertEqual(circuit.layout, new_circuit.layout)
```